### PR TITLE
Update Minimap2 to version 2.1.r311

### PR DESCRIPTION
**Pull request by Valiec**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/minimap2/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "2.1.r311" %}
+{% set hash = "ee47029535945ca595272e4c8c1ab4bad1b652350fd6d6fd7befc09a57b9d85a" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/lh3/minimap2/archive/5c4d040b13f6aa714696605f3cf618ad5b1f934e.tar.gz
+  url: https://github.com/lh3/minimap2/archive/v2.1.tar.gz
```
Comment (peterjc) [inline recipes/minimap2/meta.yaml]: Stray trailing whitespace

**Comment by peterjc** [comment]
I would personally also add this to ``meta.yaml``, under the line stating the license is MIT:

```
  license_file: LICENSE.txt
```

**Comment by peterjc** [comment]
Other than those two minor issues, this looks good to me.

**Comment by Valiec** [comment]
I fixed the issues.

**Comment by peterjc** [comment]
Thank you!

**Pull request merged**